demisto_sdk/__main__.py

```diff
@@ -194,12 +195,22 @@ def main(ctx, config, version, release_notes, **kwargs):
         log_file_path=kwargs.get("log_file_path"),
     )
     global logger
+
     logger = logging.getLogger("demisto-sdk")
     handle_deprecated_args(ctx.args)
 
     config.configuration = Configuration()
     import dotenv
 
+    # make sure that git is installed before using any command.
+    try:
+        subprocess.check_output(["git", "--version"])
```

Comment:
See GitUtil, we (re)raise `InvalidGitRepositoryError`, better handle it there.

Reply:
however, we have several places that we just the `Repo` class from github-python and not the `GitUtil` object
In order words we do not use only the `GitUtil` class for git stuff.

Reply:
basically it crashes when the first `git` imported is reached (which happens in `__main__.py`)